### PR TITLE
Adding surge link ( auto generation of PR repo Links )

### LINKED ISSUE #832: Use Surge to generate working demo from each PR?
Wow, surge is pretty cool!  https://surge.sh/

@subhahu123 has opened this PR on the `community-toolbox` project to auto-generate a static site from each PR. We could potentially have this generate a working version of `image-sequencer` based on each PR... what do you think? 

https://github.com/publiclab/community-toolbox/pull/147/checks?check_run_id=71695437#

### PR DESCRIPTION
**Pull request by subhahu123**
Fixes #832

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
I checked the travis log. You will have to change the file permissions by running `chmod +x pr_deploy.sh` if you are using UNIX based system. If you are using windows you will have to search for a workaround on the internet. Thanks.

**Comment by subhahu123** [comment]
> I checked the travis log. You will have to change the file permissions by running `chmod +x pr_deploy.sh` if you are using UNIX based system. If you are using windows you will have to search for a workaround on the internet. Thanks.

i have done this already two time in my linux but when i upload the file to repo it shows error
i m not using git for this .. directly  uploading the 
i have also give full permission like everything chmod 777 pr_deploy.sh but it still shows error on it

**Comment by HarshKhandeparkar** [comment]
I think sudo will probably fix this(travis hasn't passed yet). I also don't think you should give it 777 permissions. I think 771 or 775 would be better.

**Comment by subhahu123** [comment]
> I think sudo will probably fix this(travis hasn't passed yet). I also don't think you should give it 777 permissions. I think 771 or 775 would be better.

i have already changed the travis file
Adding sudo: required at the top let see it work or nt

**Comment by subhahu123** [comment]
> I think sudo will probably fix this(travis hasn't passed yet). I also don't think you should give it 777 permissions. I think 771 or 775 would be better.

does the method matter of uploading the file like using git or directly uploading the file

**Comment by HarshKhandeparkar** [comment]
Hmm, it failed. Try putting sudo while running the command `sudo ./pr_deploy.sh`. Btw does it work offline?

**Comment by HarshKhandeparkar** [comment]
> > I think sudo will probably fix this(travis hasn't passed yet). I also don't think you should give it 777 permissions. I think 771 or 775 would be better.
> 
> does the method matter of uploading the file like using git or directly uploading the file

I don't think it should. Finally all the git clients and code editors use git in the background, they just abstract it away

**Comment by subhahu123** [comment]
> > > I think sudo will probably fix this(travis hasn't passed yet). I also don't think you should give it 777 permissions. I think 771 or 775 would be better.
> > 
> > 
> > does the method matter of uploading the file like using git or directly uploading the file
> 
> I don't think it should. Finally all the git clients and code editors use git in the background, they just abstract it away

then why is travis failing

**Comment by HarshKhandeparkar** [comment]
Try running sudo ./pr_deploy.sh. Did travis pass in the community-toolbox repo?

**Comment by HarshKhandeparkar** [comment]
Maybe you need to run `chmod +x ./pr_deploy.sh` in travis as well?

**Comment by subhahu123** [comment]
> Try running sudo ./pr_deploy.sh. Did travis pass in the community-toolbox repo?

Yes u can check

**Comment by subhahu123** [comment]
> Maybe you need to run `chmod +x ./pr_deploy.sh` in travis as well?

i don't think so

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +17,7 @@ script:
   - npm test
   - npm run test-ui
   - grunt build
```
Comment (HarshKhandeparkar) [inline .travis.yml]: Do you have to install surge first? `npm i -g surge`??

Comment (subhahu123) [reply .travis.yml]: surge is just pushing the code to server it is independent of this project

Comment (subhahu123) [reply .travis.yml]: it only push or uplad the files

Comment (subhahu123) [reply .travis.yml]: it is working perfectly in community - toolbox

**Comment by subhahu123** [comment]
> As a last try, you can try running `sudo ./pr_deploy.sh` and running `sudo chmod +x ./pr_deploy.sh` in travis. What do you think?

ok

**Comment by subhahu123** [comment]
> As a last try, you can try running `sudo ./pr_deploy.sh` and running `sudo chmod +x ./pr_deploy.sh` in travis. What do you think?

let's see ????

**Comment by HarshKhandeparkar** [comment]
It failed but it says `surge command not found`

**Pull request closed without merging**